source/examples/generated/dotnet-flexible-sync/FlexibleSyncExamples.snippet.open-fs-realm.cs

```diff
@@ -0,0 +1,12 @@
+var app = App.Create("myRealmAppId");
+var user = await app.LogInAsync(Credentials.Anonymous());
+
+var config = new FlexibleSyncConfiguration(app.CurrentUser!)
+{
+    PopulateInitialSubscriptions = (realm) =>
+    {
+        var allTasks = realm.All<MyTask>();
+        realm.Subscriptions.Add(allTasks, new SubscriptionOptions { Name = "allTasks" });
+    }
+};
+var realm = Realm.GetInstance(config);
```

Comment:
```suggestion
var realm = await Realm.GetInstanceAsync(config);
```
In the text preceding this we say we're going to use `GetInstanceAsync` so that's what we should use. 
Also, in the following snippet, that I think is not included, we are using `GetInstance`, while we say the opposite. We should fix that snippet too
(in the "Configure & Open a Synced Realm" page)